src/Ledger/NewPP.lagda

```diff
@@ -24,7 +24,7 @@ record NewPParamState : Set where
 
 updatePPUp : PParams → PPUpdateState → PPUpdateState
 updatePPUp pparams record { fpup = fpup }
-  with allᵇ (isViableUpdate? pparams) (range fpup)
+  with allᵇ ¿ isViableUpdate pparams ¿¹ (range fpup)
```

Comment:
This is the only change that leaks to the user (since `isViableUpdate?` was completely redundant).

Anyway, this related to #281 so let's solve this in a separate PR that fixes this particular issue.